packages/components/multiselect/README.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Multiselect'
+slug: /components/multiselect/
+github: 'https://github.com/contentful/forma-36/tree/main/packages/components/multiselect'
+storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-multiselect--basic'
+typescript: ./src/Multiselect.tsx
+---
+
+Search through a long list of options.
+
+## Import
+
+```jsx static=true
+import { Multiselect } from '@contentful/f36-components';
+// or
+import { Multiselect } from '@contentful/f36-multiselect';
+```
+
+## Examples
+
+### Basic usage
+
+The Multiselect requires 3 props to work:
+
+- `items`: It’s an array containing the items that will be shown as selectable options when the user types something in the `TextInput`.
+- `onInputValueChange`: This function will be called every time the user types something in the input.
```

Comment:
Since this is a public interface, I would abstract some internal knowledge away to make it more smooth/ easier to read.
For the naming, my proposal would be `onSearchValueChange` since the users types something to search for stuff - while input is a very general name. The MultiSelect itself is an input.